src/main/java/org/betonquest/betonquest/modules/web/updater/source/implementations/GitHubReleaseSource.java

```diff
@@ -14,31 +15,54 @@
 /**
  * This is a {@link ReleaseUpdateSource} for GitHub's releases API.
  */
-public class GitHubReleaseSource extends UpdateSource implements ReleaseUpdateSource {
+public class GitHubReleaseSource implements ReleaseUpdateSource {
     /**
      * The sub path of the release API of GitHub to append on the {@link GitHubReleaseSource#apiUrl}.
      */
     public static final String RELEASES_URL = "/releases";
+
+    /**
+     * Response code 403 forbidden is returned by GitHub if too many requests are made.
+     */
+    /* default */ static final int RESPONSE_CODE_403 = 403;
```

Comment:
This is the second badly named HTTP response code constant...
How about a general static `ResponseCode` utility class that just contains constants for HTTP response codes? (Not an enum though, response codes aren't a closed set!)